modules/assim.batch/R/pda.bayestools.helpers.R

```diff
@@ -0,0 +1,233 @@
+##' Create priors for BayesianTools
+##' 
+##' Helper function for creating log-priors compatible with BayesianTools package
+##'
+##' @param prior.sel `data.frame` containing prior distributions of the selected parameters
```

Comment:
Not required but could have a data type check if it needs a data.frame with dist, a, b in case something else is passed